app/controllers/manuals_controller.rb

```diff
@@ -107,4 +75,29 @@ def set_expiry(content_item)
     cache_private = content_item.cache_control.private?
     expires_in(max_age, public: !cache_private)
   end
+
+  def follow_redirect_if_document_redirected
```

Comment:
Is this new functionality? If so, could it be its own separate commit?

Reply:
No new functionality, just a slightly different name. The old logic resided in a [different method](https://github.com/alphagov/manuals-frontend/pull/1113/files#diff-6b5ff7cba3fc3659ddf2782828d61b92542869a958e4970b0d650d108ba881c0L52-L54).